manim/plugins/import_plugins.py

```diff
@@ -0,0 +1,41 @@
+import pkg_resources
+from importlib import import_module
+from .. import config, logger
+
+__all__ = []
+
+# don't know what is better to do here!
+module_type = type(pkg_resources)
+function_type = type(import_module)
```

Comment:
You can import `types` and then use `types.ModuleType` and `types.FunctionType` instead of defining these yourself